AK/Slugify.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2019-2020, Gurkirat Singh <tbhaxor@gmail.com>
```

Comment:
This year is 2023.

Reply:
Yeah, I copied it from different file. 😆